packages/api/src/context.ts

```diff
@@ -448,26 +454,30 @@ export async function contextFromRequest(
               },
               OR: [
                 {
-                  published: {
```

Comment:
when introducing a n2n table in the revisions, the prisma interface changed from

```
export type ArticleRevisionRelationFilter = {
    is?: ArticleRevisionWhereInput | null
    isNot?: ArticleRevisionWhereInput | null
}
```

to 

```
export type ArticleRevisionRelationFilter = {
    is?: ArticleRevisionWhereInput
    isNot?: ArticleRevisionWhereInput
}
```